mautic/mautic
# Discrepancy report

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Create a report of discrepancies of data synced across integrations with Mautic

#### Steps to test this PR:
1. Apply PR run `app/console mautic:integration:discrepancies --integration Salesforce --env dev --time-interval='10 days' -m sendto@email.com`

2. you should get a csv file of discrepancies report
